samples/frameworks/react/ts/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
probably we can target a lot newer than this, given your browser targets in the package.json. Maybe even you could target a runtime that natively supports `async await` to avoid having to transpile it!